### PR TITLE
Updgrade Lottie on Android and iOS

### PR DESCRIPTION
**Pull request by SalvatoreT**
# Summary

The linked version of Lottie are pretty old, and this fixes that.

### What's required for testing (prerequisites)?

Someone needs to run this on Android and iOS. I would if I knew how. I opened an issue for the lack of documentation around getting started. https://github.com/react-native-community/lottie-react-native/issues/650

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅     |
| Android |    ✅     |

## Checklist

- [X] I have tested this on a device and a simulator
- [X] I added the documentation in `README.md`
- [X] I mentioned this change in `CHANGELOG.md`
- [ ] ~I updated the typed files (TS and Flow)~
- [ ] ~I added a sample use of the API in the example project (`example/App.js`)~

### CODE REVIEW

**Comment by emilioicai** [comment]
@SalvatoreT thanks for this PR, I have trouble building on a simulator. Did this work for you?

**Comment by SalvatoreT** [comment]
@emilioicai, I need to land this PR first which fixes the Android build. https://github.com/react-native-community/lottie-react-native/pull/652

**Comment by emilioicai** [comment]
I was just reviewing https://github.com/react-native-community/lottie-react-native/pull/652 , it should be merged now :)

**Comment by emilioicai** [comment]
@SalvatoreT any luck having these changes working on a device/simulator?

**Comment by SalvatoreT** [comment]
I can run the changes on an Android simulator and device, but I can't run the iOS example on this branch or `master`.

**Comment by SalvatoreT** [comment]
I made a PR to upgrade and fix the development environment to unblock this. https://github.com/react-native-community/lottie-react-native/pull/653

**Comment by SalvatoreT** [comment]
I rebased this branch on the one from #653 and it worked! 🎉

I can rebase this branch once/if #653 lands on `master`.

**Comment by emilioicai** [comment]
Hey, thanks for all the effort! https://github.com/react-native-community/lottie-react-native/pull/653 has landed on master, so you can rebase when you've got the chance

**Comment by SalvatoreT** [comment]
Rebased and ran through Android and iOS again. ✅

Screen Shot 2020-05-21 at 11 06 52 AM

Screen Shot 2020-05-21 at 11 08 03 AM

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
   },
   "dependencies": {
     "invariant": "^2.2.2",
-    "lottie-ios": "^3.1.3",
+    "lottie-ios": "3.1.6",
```
Comment (emilioicai) [inline package.json:50]: shouldn't this be 3.1.8?

Comment (SalvatoreT) [reply package.json:50]: 🤦 yep. I'll get right on it.

**Comment by SalvatoreT** [comment]
Fixed!

**Comment by emilioicai** [comment]
Thanks for the good work @SalvatoreT !

**Comment by SalvatoreT** [comment]
You're welcome!

**Comment by emilioicai** [comment]
Since this PR includes some documentation changes I will merge once we have a new version ready so users are not confused.

**Comment by Eyesonly88** [comment]
Awesome work @SalvatoreT 👏 
@emilioicai any ETA on the new version? really looking forward to updating the dependencies.

**Comment by emilioicai** [comment]
Hopefully this PR should land next week on a new version

**Comment by lucianomlima** [comment]
Any news about the new version release?

**Comment by ninjz** [comment]
I see this has been approved. Any reason why this is not being merged yet @emilioicai ?

**Pull request merged**